src/platform/packages/shared/kbn-esql-validation-autocomplete/scripts/generate_function_definitions.ts

```diff
@@ -432,7 +426,6 @@ export const esqlFunctionNames = ${JSON.stringify(functionNames, null, 2)};
     }
 
     const functionDefinition = getFunctionDefinition(ESDefinition);
-    const isLikeOperator = functionDefinition.name.toLowerCase().includes('like');
```

Comment:
Like is correctly categorized as operator, this distinction was not necessary here